database.json

```diff
@@ -2676,6 +2682,12 @@
     "repo": "typescript",
     "desc": "TypeScript is a superset of JavaScript that compiles to clean JavaScript output."
   },
+  "twitter-lite-deno": {
+    "type": "github",
+    "owner": "alkihis",
+    "repo": "twitter-lite-deno",
+    "desc": "Port of Twitter Lite library to Deno, a tiny, modern and full-featured library for the Twitter API."
+  },
```

Comment:
database entries should be sorted alphabetically. Please insert your new entry in the correct location.